bcda/models/models.go

```diff
@@ -286,22 +286,15 @@ func CreateUser(name string, email string, acoUUID uuid.UUID) (User, error) {
 }
 
 // CLI command only support; note that we are choosing to fail quickly and let the user (one of us) figure it out
-func CreateAlphaACO(db *gorm.DB) (ACO, error) {
+func CreateAlphaACO(acoCMSID string, db *gorm.DB) (ACO, error) {
```

Comment:
Correct me if I'm wrong, but it seems like the only thing that calls this code is `createAlphaEntities()`, so this could go as well.